lua/nvim_lsp/configs.lua

```diff
@@ -46,6 +46,14 @@ function configs.__newindex(t, config_name, config_def)
       end
     end
 
+    config.callbacks["window/showMessage"] = function(err, method, params, client_id)
+      if params and params.type <= config.message_level then
+        -- TODO(ashkan) remove this after things have settled.
+        assert(lsp.callbacks, "Update to Nvim HEAD. This is an incompatible interface.")
+        lsp.callbacks[method](err, method, params, client_id)
```

Comment:
what kind of strings will `method` be? should we validate it ? if `lsp.callbacks[method]` is potentially nil, we should show a meaningful error message .

Reply:
I fix it.